### PR TITLE
Update installation.rst

### PR DESCRIPTION
**Pull request by DavidGill159**
Lightning Pose installation instructions added for Windows users.

### CODE REVIEW

File: docs/source/installation.rst [hunk 1]
```diff
@@ -17,6 +17,8 @@ There are several installation methods:
 
 Optionally, instructions are provided for :ref:`Docker users <docker_users>`.
 
+If you are a **Windows user**, please read `Windows Installation with WSL`_ first.
```
Comment (themattinthehatt) [inline docs/source/installation.rst:20]: You'll have to update the internal reference here to point to the windows section; to do this you can follow the Docker example. You'll need to replace 
`Windows Installation with WSL`_ 
with 
:ref:`Windows Installation with WSL <windows_users>`
The "<windows_users>" part is the actual tag for where the link should go.
You can then add that tag above the windows section:
.. _windows_users:
(note the underscore _before_ windows, this is required by RST files)

File: docs/source/installation.rst [hunk 2]
```diff
@@ -162,6 +164,85 @@ memory to execute.
 The ``--gpus all`` flag is necessary to allow Docker to access the required drivers for NVIDIA DALI to work properly.
 
 
+
```
Comment (themattinthehatt) [inline docs/source/installation.rst:167]: add
.. _windows_users:
here (see the similar pattern above the "Docker users" heading)

File: docs/source/installation.rst [hunk 3]
```diff
@@ -162,6 +164,85 @@ memory to execute.
...
+    wsl.exe --install
+
+3. Ensure you have the latest WSL kernel by running: 
+
+.. code-block:: console
+
+    wsl.exe --update
+
+4. Restart your computer. This is necessary for WSL Ubunta to take full effect.
```
Comment (themattinthehatt) [inline docs/source/installation.rst:200]: Ubunta -> Ubuntu?

**Review by themattinthehatt — CHANGES_REQUESTED** [review]
Thanks for the thorough instructions @DavidGill159! We very much appreciate the help. I left a couple comments, the main one is about creating an internal link. After that should be good to go.

**Comment by DavidGill159** [comment]
Hi Matt, I have made the requested changes.

**Review by themattinthehatt — APPROVED** [review]

**Pull request merged**